Cargo.toml

```diff
@@ -17,6 +17,7 @@ resolver = "2"
 mp3 = ["kira/mp3"]
 ogg = ["kira/ogg"]
 flac = ["kira/flac"]
+spec = ["ron", "serde"]
```

Comment:
I wasn't quite sure what to call this feature. spec doesn't really describe what it does, let me know if you have any suggestions.

Reply:
Maybe "settings_loader" or just "settings"? Not sure...

Reply:
I think settings_loader is probably clearer.